packages/protocol/test-sol/utils/ECDSAHelper.sol

```diff
@@ -13,7 +13,7 @@ contract ECDSAHelper is Test {
     bytes32 _s
   ) public returns (bytes memory) {
     address SECP256K1Address = actor("SECP256K1Address");
-    deployCodeTo("SECP256K1.sol:SECP256K1", SECP256K1Address);
+    deployCodeTo("out/SECP256K1.sol/SECP256K1.0.5.17.json", SECP256K1Address);
```

Comment:
This seems to be never ending story

Reply:
Unfortunately, I had to downgrade the foundry version and revert this change (from https://github.com/celo-org/celo-monorepo/pull/10997), because of an unrelated regression in the latest foundry version. 

When the regression is fixed and we can bump the version, I don't expect this to be a problem again.